test/e2e/flask/multichain-api/invoke-method.spec.ts

```diff
@@ -0,0 +1,274 @@
+            );
+
+            const scopeCards = await driver.findElements('.scope-card');
+            for (const i of scopeCards.keys()) {
+              const scope = GANACHE_SCOPES[i];
+              await driver.clickElementSafe(
+                `[data-testid="${scope}-eth_sendTransaction-option"]`,
+              );
+
+              i === INDEX_FOR_ALTERNATE_ACCOUNT &&
+                (await driver.clickElementSafe(
+                  `[data-testid="${scope}:${ACCOUNT_2}-option"]`,
+                ));
+            }
+
+            await driver.clickElement({
+              text: 'Invoke All Selected Methods',
+              tag: 'button',
+            });
+
+            for (const i of GANACHE_SCOPES.keys()) {
+              await driver.delay(largeDelayMs);
+              await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
+
+              const accountWebElement = await driver.findElement(
+                '[data-testid="sender-address"]',
+              );
+              const accountText = await accountWebElement.getText();
+              const expectedAccount =
+                i === INDEX_FOR_ALTERNATE_ACCOUNT ? 'Account 2' : 'Account 1';
+
+              assert.strictEqual(
+                accountText,
+                expectedAccount,
+                `Should have ${expectedAccount} selected, got ${accountText}`,
+              );
+
+              await driver.clickElement({
+                text: 'Confirm',
+                tag: 'button',
+              });
+            }
+          },
+        );
+      });
+
+      it('should have less balance due to gas after transaction is sent', async function () {
+        await withFixtures(
+          {
+            title: this.test?.fullTitle(),
+            fixtures: new FixtureBuilder()
+              .withNetworkControllerTripleGanache()
+              .build(),
+            ...DEFAULT_MULTICHAIN_TEST_DAPP_FIXTURE_OPTIONS,
+          },
+          async ({
+            driver,
+            extensionId,
+          }: {
+            driver: Driver;
+            extensionId: string;
+          }) => {
+            await openMultichainDappAndConnectWalletWithExternallyConnectable(
+              driver,
+              extensionId,
+            );
+            await initCreateSessionScopes(driver, GANACHE_SCOPES, ACCOUNTS);
+            await addAccountInWalletAndAuthorize(driver);
+            await driver.clickElement({ text: 'Connect', tag: 'button' });
+
+            await driver.delay(largeDelayMs);
+            await driver.switchToWindowWithTitle(
+              WINDOW_TITLES.MultichainTestDApp,
+            );
+
+            const scopeCards = await driver.findElements('.scope-card');
+
+            for (const i of scopeCards.keys()) {
+              const scope = GANACHE_SCOPES[i];
+              await driver.clickElementSafe(
+                `[data-testid="${scope}-eth_sendTransaction-option"]`,
+              );
+
+              i === INDEX_FOR_ALTERNATE_ACCOUNT &&
```

Comment:
thoughts on using something like `i % 2` instead? Idk. no strong opinion on this one

Reply:
ehh nevermind. What you have is good